dsc/src/subcommand.rs

```diff
@@ -320,14 +320,15 @@ pub fn resource(subcommand: &ResourceSubCommand, stdin: &Option<String>) {
         ResourceSubCommand::List { resource_name, description, tags, format } => {
 
             let mut write_table = false;
-            let mut table = Table::new(&["Type", "Version", "Requires", "Description"]);
+            let mut methods = String::new();
```

Comment:
I think it would be better to have this as a vec of strings which has a `.join()` function you can use